app/widgets/common/filters/inputs/AttributeValueInput.tsx

```diff
@@ -27,7 +27,7 @@ const AttributeValueInput: FunctionComponent<AttributeValueInputProps> = ({
             <DynamicDropdown
                 name="ruleValue"
                 fetchUrl={fetchUrl}
-                onChange={onChange}
+                onChange={onChange as DynamicDropdownProps['onChange']}
```

Comment:
Optional:

```suggestion
                onChange={ruleValue => onChange(ruleValue as string[])}
```

I'm wondering would it be possible to define `DynamicDropdown` props in such a way that when `mutliple` is set to true we are sure that `value` is `string[] | undefined` and `onChange` as it's first argument gets `string[] | undefined`. Probably with use of discriminated union it could be possible. That's of course not for now - probably for RD-4546.

Reply:
Oh, that's an interesting idea
For sure it would simplify some things and would made our lives easier 😁 

First thing that is coming to my mind, would be to extract 2 components:
- SingleValueDropdown
- MultipleValueDropdown
(which could be named differently 😁)
which would have `onChange` typed differently - but it's seems like an overkill 😄 

Having differently typed `onChange` (depending on the `multiple` prop) seems like a great if not ideal solution 😁 
I'll try to research that a little bit - if I'd get some satisfactory answer, I'll let you know 💪 
If you'd also get any kind of idea about the implementation of that solution, don't hesitate to share it 😁

Reply:
Suggested modification has been applied in 4aa59fa0f47ead27aa9b942a0a905baf18da796f 🎉